.github/workflows/publish_charm.yaml

```diff
@@ -0,0 +1,13 @@
+name: Publish to edge
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  test-and-publish-charm:
```

Comment:
Suggestion: this can be renamed to `publish-charm` if no testing is being conducted.

Reply:
Out of curiosity, what is the reason you no longer want to test on publishing of the charm? I've found this useful when there have been changes to depending charms between submitting the PR for review and the merge.

Reply:
The operator workflows changed a while back to this approach, where tests are no longer needed once a PR is merged. There's a point to be made about the need to run the test again before publishing, but I opted to use the workflow as is and adapt to any changes needed.